### PR TITLE
Prefer `main` over `master` everywhere it makes sense

### PR DESCRIPTION
**Pull request by pda**
Use `main` rather than `master` anywhere that makes sense; mostly in example API requests and responses.

Note that the _default_ default branch for a pipeline is now `main` not `master`, so that particular update is required anyway.

https://github.com/buildkite/elastic-ci-stack-for-aws is still using `master` for now (moving is tricky, but we'll get there) so those references haven't been updated.

Related:
- https://sfconservancy.org/news/2020/jun/23/gitbranchname/

### CODE REVIEW

File: data/graphql_data_schema.json [hunk 1]
```diff
@@ -22161,7 +22161,7 @@
                 "name": "String",
                 "ofType": null
               },
-              "defaultValue": "\"master\"",
+              "defaultValue": "\"main\"",
```
Comment (pda) [inline data/graphql_data_schema.json:22164]: True fact, `main` is the new default default.

File: pages/agent/v2/docker.md [hunk 2]
```diff
@@ -29,4 +29,4 @@ By default Docker [starts containers on restart](https://docs.docker.com/article
 
 ## SSH keys, configuration and customization
 
-See the [GitHub repository](https://github.com/buildkite/docker-buildkite-agent) for instructions on how to customize the base image. Alternatively you can create your own Docker image using our standard installer packages (see our [Ubuntu Dockerfile](https://github.com/buildkite/agent/blob/master/packaging/docker/ubuntu-linux/Dockerfile) for an example).
+See the [GitHub repository](https://github.com/buildkite/docker-buildkite-agent) for instructions on how to customize the base image. Alternatively you can create your own Docker image using our standard installer packages (see our [Ubuntu Dockerfile](https://github.com/buildkite/agent/blob/main/packaging/docker/ubuntu-22.04/Dockerfile) for an example).
```
Comment (pda) [inline pages/agent/v2/docker.md:32]: The old URL was broken anyway, this fixes it.

**Review by DrJosh9000 — APPROVED** [review]
Yessssss 🤩

**Review by mbelton-buildkite — COMMENTED** [review]
Nice! Thanks for updating all of these! There's one screenshot that's out of sync, so it would be good to update that alongside the text. 😊

**Comment by mbelton-buildkite** [comment]
@pda, I'm skimming through old PRs to see if there are any that can be quickly resolved, and this looks like a great candidate. From what I can see, it's just hung up on a screenshot. I'll update the branch, fix the screenshot, and merge. Let me know if there are any issues with that. 😊

File: pages/pipelines/conditionals.md [hunk 3]
```diff
@@ -7,9 +7,7 @@ You can define conditionals at the step level in your `pipeline.yml` or at the p
 
 ## Conditionals in pipelines
 
-<!--alex ignore master-->
-
-You can have complete control over when to trigger pipeline builds by using conditional expressions to filter incoming webhooks. You need to define conditionals in the _Pipeline Settings page_ for your repository provider to run builds only when expressions evaluate to `true`. For example, to run only when a pull request is targeting the master branch:
+You can have complete control over when to trigger pipeline builds by using conditional expressions to filter incoming webhooks. You need to define conditionals in the _Pipeline Settings page_ for your repository provider to run builds only when expressions evaluate to `true`. For example, to run only when a pull request is targeting the main branch:
```
[the remark this answers is not in the document]
Comment (mbelton-buildkite) [reply pages/pipelines/conditionals.md:10]: ✅ Done

**Review by mbelton-buildkite — APPROVED** [review]

**Comment by pda** [comment]
Thanks very much @mbelton-buildkite 🙏🏼

**Pull request merged**